dbloader/mongo_loader.py

```diff
@@ -46,7 +46,7 @@ def insert(self, database, collection, custom=None):
             self.get_connection()
         db = self.conn[database]
         try:
-            random_text = Loader.big_string(100)
+            random_text = self.big_string(self.string_size)
```

Comment:
Just fyi, static methods can be called using class as well as instance. So all three options: 'Loader.big_string', 'MongoLoader.big_string' as well as 'self.big_string' are valid :-)